R/lm_robust.R

```diff
@@ -25,7 +25,9 @@ lm_robust <- function(formula,
                       ci = TRUE,
                       alpha = .05,
                       coefficient_name = NULL,
-                      return_vcov = TRUE) {
+                      return_vcov = TRUE,
+                      ei = TRUE,
+                      trychol = FALSE) {
 
```

Comment:
FYI, ei means use Eigen and `lm_speed.cpp` versus the old algorithm in `lm_robust_helper.cpp` (i.e. armadillo `solve(Xt*X)`).

`trychol` lets `lm_sped.cpp` know whether it should try the cholesky decomposition first.